src/plugins/workspace/server/plugin.ts

```diff
@@ -155,7 +156,8 @@ export class WorkspacePlugin implements Plugin<{}, {}> {
           features: [
             `@${DEFAULT_APP_CATEGORIES.management.id}`,
             WORKSPACE_OVERVIEW_APP_ID,
-            WORKSPACE_UPDATE_APP_ID,
```

Comment:
I think `WORKSPACE_UPDATE_APP_ID` should not be removed, we should allow people to update management workspace settings

Reply:
Should **Workspace settings** be removed for our initial release?